plugin/converter/starlark/starlark.go

```diff
@@ -146,12 +128,10 @@ func (p *starlarkPlugin) Convert(ctx context.Context, req *core.ConvertArgs) (*c
 	// this is a temporary workaround until we
 	// implement a LimitWriter.
 	if b := buf.Bytes(); len(b) > limit {
-		return nil, ErrMaximumSize
+		return nil, nil
```

Comment:
why was the return `ErrMaximumSize` removed?

Reply:
Not sure how i missed that, fixed.